javascript/src/main/pages/TutorAssignment/TutorAssignments.js

```diff
@@ -34,7 +34,7 @@ const TutorAssignment = () => {
   if (error) {
     return (
       <>
-        {isInstructor && <Button onClick={()=>history.push("/tutorAssignments/new")}>New Tutor Assignment</Button>}
+        {isInstructor && <Button style={{marginBottom: "1em"}} onClick={()=>history.push("/tutorAssignments/new")}>New Tutor Assignment</Button>}
```

Comment:
I would strongly recommend using Bootstrap's built-in utilities/components instead of using custom css when possible. This can be replaced with something like `className="mb-3"` ([docs](https://getbootstrap.com/docs/4.0/utilities/spacing/)).